README.md

```diff
@@ -18,7 +18,10 @@ Cocotb requires:
 
 - Python 3.5+
 - A C++11 compiler
-- An HDL simulator (such as [Icarus Verilog](http://iverilog.icarus.com/))
+- An HDL simulator (such as [Icarus Verilog](http://iverilog.icarus.com/),
+[Verilator](https://www.veripool.org/wiki/verilator),
+[GHDL](http://ghdl.free.fr/) or a
+[commercial simulator](https://docs.cocotb.org/en/stable/simulator_support.html))
```

Comment:
```suggestion
[another simulator](https://docs.cocotb.org/en/stable/simulator_support.html))
```
since importantly this page describes the limitations of _all_ simulators.